src/modm/platform/can/stm32/can.cpp.in

```diff
@@ -269,9 +271,8 @@ MODM_ISR({{ reg }}_RX0)
 	// Release FIFO (access the next message)
 	{{ reg }}->RF0R = CAN_RF0R_RFOM0;
 
-	if (!rxQueue.push(rxMessage)) {
-		modm::ErrorReport::report(modm::platform::CAN{{ id }}_FIFO0_OVERFLOW);
-	}
+	modm_assert_continue_ignore(rxQueue.push(rxMessage), "can.rx.sw0",
+		"CAN receive software buffer overflowed!", {{ 0 if id == '' else id }});
```

Comment:
Have we ever used this ErrorReport in the RCA hardware? cc @rleh @se-bi

Reply:
I don't think so.